omod/src/main/java/org/bahmni/module/bahmni/ie/apps/controller/BahmniFormController.java

```diff
@@ -10,71 +11,81 @@
-	}
+    @RequestMapping(value = baseUrl + "/translate", method = RequestMethod.GET)
+    @ResponseBody
+    public FormFieldTranslations translate(@RequestParam(value = "formName") String formName,
+                                           @RequestParam(value = "formVersion") String formVersion,
+                                           @RequestParam(value = "locale") String locale) {
+        return bahmniFormTranslationService.setNewTranslationsForForm(locale, formName, formVersion);
+    }
 
-	@RequestMapping(value = baseUrl + "/publish", method = RequestMethod.POST)
-	@ResponseBody
-	public BahmniForm publish(@RequestParam("formUuid") String formUuid) {
-		return bahmniFormService.publish(formUuid);
-	}
+    @RequestMapping(value = baseUrl + "/publish", method = RequestMethod.POST)
+    @ResponseBody
+    public BahmniForm publish(@RequestParam("formUuid") String formUuid) {
+        return bahmniFormService.publish(formUuid);
+    }
 
-	@RequestMapping(value = baseUrl + "/save", method = RequestMethod.POST)
-	@ResponseBody
-	public BahmniFormResource save(@RequestBody BahmniFormResource bahmniFormResource) {
-		return bahmniFormService.saveFormResource(bahmniFormResource);
-	}
+    @RequestMapping(value = baseUrl + "/save", method = RequestMethod.POST)
+    @ResponseBody
+    public BahmniFormResource save(@RequestBody BahmniFormResource bahmniFormResource) {
+        return bahmniFormService.saveFormResource(bahmniFormResource);
+    }
 
-	@RequestMapping(value = baseUrl + "/latestPublishedForms", method = RequestMethod.GET)
-	@ResponseBody
-	public List<BahmniForm> getLatestPublishedForms(
-			@RequestParam(value = "includeRetired", defaultValue = "false") boolean includeRetired,
-			@RequestParam(value = "encounterUuid", required = false) String encounterUuid) {
-		return bahmniFormService.getAllLatestPublishedForms(includeRetired, encounterUuid);
-	}
+    @RequestMapping(value = baseUrl + "/latestPublishedForms", method = RequestMethod.GET)
+    @ResponseBody
+    public List<BahmniForm> getLatestPublishedForms(
+            @RequestParam(value = "includeRetired", defaultValue = "false") boolean includeRetired,
+            @RequestParam(value = "encounterUuid", required = false) String encounterUuid) {
+        return bahmniFormService.getAllLatestPublishedForms(includeRetired, encounterUuid);
+    }
 
-	@RequestMapping(value = baseUrl + "/allForms", method = RequestMethod.GET)
-	@ResponseBody
-	public List<BahmniForm> getAllForms() {
-		return bahmniFormService.getAllForms();
-	}
+    @RequestMapping(value = baseUrl + "/allForms", method = RequestMethod.GET)
+    @ResponseBody
+    public List<BahmniForm> getAllForms() {
+        return bahmniFormService.getAllForms();
+    }
+
+    @RequestMapping(value = baseUrl + "/export", method = RequestMethod.GET)
+    @ResponseBody
+    public ExportResponse export(@RequestParam ("uuids") List<String> uuids) {
```

Comment:
Has this change also be made on the UI?

Reply:
On UI, request api is calling with params, wherein params are made of the object UUIDs.

Reply:
it supposed to be uuid only, as we are passing values with keyword 'uuid'. it will collect all values with that key word and receive them as a list.

Reply:
i will make changes and push